src/Adapters.Tests/NLogTarget.Tests.Shared/NLogTargetTests.cs

```diff
@@ -232,12 +232,31 @@ public void TraceAreEnqueuedInChannelAndContainExceptionMessage()
                 aiLogger.Debug(exception, "testing exception scenario");
             }
 
-            var telemetry = (ExceptionTelemetry)this.adapterHelper.Channel.SentItems.FirstOrDefault();
+            var telemetry = (ExceptionTelemetry)this.adapterHelper.Channel.SentItems.First();
             Assert.AreEqual(expectedException.Message, telemetry.Exception.Message);
         }
 
         [TestMethod]
         [TestCategory("NLogTarget")]
+        public void CustomMessageIsAddedToExceptionTelemetryCustomProperties()
+        {
+            Logger aiLogger = this.CreateTargetWithGivenInstrumentationKey();
+            
+            try
+            {
+                throw new Exception("Test logging exception");
```

Comment:
We want to have a real exception with stack.

---
In reply to: [66859585](https://github.com/Microsoft/ApplicationInsights-dotnet-logging/pull/36#discussion_r66859585) [](ancestors = 66859585)